packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -242,7 +242,15 @@ class FlutterPlugin implements Plugin<Project> {
 
     private Properties getPluginList(Project project) {
         File pluginsFile = new File(project.projectDir.parentFile.parentFile, '.flutter-plugins')
-        return readPropertiesIfExist(pluginsFile)
+        Properties allPlugins = readPropertiesIfExist(pluginsFile)
+        Properties androidPlugins = new Properties()
+        allPlugins.each { name, path ->
+            File editableAndroidProject = new File(path, 'android' + File.separator + 'build.gradle')
+            if (editableAndroidProject.exists()) {
```

Comment:
I think we might want to print some sort of warning here. Assuming that we can fix the platform specific .flutter-plugins issue, this would still be an error.

Reply:
Not sure what I can do as part of this CL...?

Reply:
print a warning?

Reply:
See: https://github.com/flutter/flutter/blob/master/packages/flutter_tools/gradle/flutter.gradle#L876

Reply:
Print a warning when? both cases (having an `android/build.gradle` and not having it are legit...)

Reply:
print a warning when it is missing. You're right for now that isn't an error because it is unavoidable. Maybe leave a todo so we can revisit it in the future.

Reply:
done